requirements.txt

```diff
@@ -1,31 +1,29 @@
-requests==2.20.0
-PyYAML==4.2b1
+django==1.11.23
 mysqlclient==1.3.13
-WebOb==1.5.0
-ecdsa==0.13.3
+requests==2.20.0
+ecdsa==0.13
 google-api-python-client==1.6.1
 httplib2==0.9.2
+oauth2client==3.0.0
 python-openid==2.2.5
 pyasn1==0.4.6
-pytz==2018.4
 requests-oauthlib==0.7.0
 rsa==3.2
 simplejson==3.8.1
+PyYAML==4.2b1
 six==1.12.0
 uritemplate==3.0.0
-pexpect==4.0.1
+GoogleAppEngineCloudStorageClient==1.9.22.1
+django-allauth==0.35.0
```

Comment:
Double check this against the WebApp's list, but I think we're on a newer version of allauth

Reply:
Changed to 0.41.0

Reply:
I removed all use of django.